### PR TITLE
Adds the Venom Antagonist

### PR DESCRIPTION
**Pull request by Fikou**
## About The Pull Request

adds the venom antagonist

dont review this its not worth it

## Why It's Good For The Game

venom they wont know what hit em
image

## Changelog

:cl:
add: Venom antag
/:cl:

### CODE REVIEW

**Comment by MrDoomBringer** [comment]
https://www.youtube.com/watch?v=8CdcCD5V-d8

**Comment by Ebin-Halcyon** [comment]
It's finally here.

File: code/__DEFINES/song.dm [hunk 1]
```diff
@@ -13,3 +13,5 @@
...
...-G-G-G-D-D-G-G-G-G/0.6,G-G-G-G-D-D-G-G-G-G\nF#-F#-F#-F#-F#-F#-F#-F#,D-D/0.6\nE4-B-A-E3-A#-G#3-D-An-B-D#2-A#-An-E-E4-Dn3-G#-B-B-D#2-A/0.6\nDn3-D/0.6,D-D/0.6,D-D/0.6,D-B-D-B/0.6,D-D/0.6\nD-D/0.6,D-D/0.6,D-B-D-B/0.6,D-D/0.6,D-D/0.6\nD5-D-D3-D5-D-D3-D5-D-D-D/1.5\nD-D-D-D-D-D-D-D/1.5\nGn5-G-G-G-D3-B-D-G-G-G-G-B/0.6\nG-G-G-G-D-D-G-G-G-G/1.5,F#-F#-F#-F#-F#-F#/1.5\nE5-E-E-D-E-D-E-E-E-E/1.5\nF#-F#-F#-F#-F#-F#-F#-F#/1.5,D-D/0.6\nA#-E3-B-E4-D-G#3-D#2-D#-B-An-E-E3-A#-G#-Dn3-B-B-An-D#2-D#/0.6\nDn3-D/0.6,D-D/0.6,D-D/0.6,D-B-D-B/0.6,D-D/0.6\nD-D/0.6,D-D/0.6\nB-E-E4-A#-G#-D-D#2-D#-B-An-E3-A#-E4-Dn3-G#/3\nB-B-D#2-D#-An/0.6,Dn3-D/0.6,D-D/0.6\nD5-D-D-D3-D5-D3-D5-D-D-D/1.5\nD-D-D-D-D-D-D-D/1.5\nGn5-G-G-D3-G-B-D-G-G-G-G-B/0.6\nG-G-G-G-D-D-G-G-G-G/0.6,G-G-G-D-G-D-G-G-G-G/1.5\nF#-F#-F#-F#-F#-F#-F#-F#/1.5,D-D/0.6,D-B-D/3\nB/0.6,D-D/0.6,D-D/0.6,D-D/0.6,D-B-D/3,B/0.6\nD-D/0.6,D-D/0.6,D-D/0.6\nE-A#-E3-B-G#3-D-D#2-An-D#-B-A#-E-E4-G#-Dn3/3\nB-B-D#2-D#-An/0.6,Dn3-D/0.6,D-D/0.6\nD5-D3-D5-D-D-D3/3,D5-D-D-D/1.5,D-D-D-D/3\nD-D-D-D/1.5,D3-Gn5-G-G-G-B-D/3,G-G-G-G-B/0.6\nG-G-D-G-G-D/3,G-G-G-G/1.5,F#-F#-F#/3\nF#-F#-F#/1.5,E5-E-D-E-E-D/3,E-E-E-E/1.5\nF#-F#-F#-F#/3,F#-F#-F#-F#/1.5,D-D/0.6,D-B-D/3\nB/0.6,D-D/0.6,D-D/0.15"
```
[the remark this answers is not in the document]
Comment (optimumtact) [reply code/__DEFINES/song.dm:17]: This song go so fast there are no new lines or pauses

**Comment by the-orange-cow** [comment]
I liked the part where you can activate the super mode and it automatically make the station announcement:  "It's Morb-venom time!"

File: code/__DEFINES/song.dm (hunk 1 above)
[the remark this answers is not in the document]
Comment (Mothblocks) [reply code/__DEFINES/song.dm:17]: this is how we're cutting donw on compile time

File: code/modules/unit_tests/simple_animal_freeze.dm [hunk 2]
```diff
@@ -302,6 +302,7 @@
 		/mob/living/simple_animal/hostile/tree,
 		/mob/living/simple_animal/hostile/tree/festivus,
 		/mob/living/simple_animal/hostile/vatbeast,
+		/mob/living/simple_animal/hostile/venom,
```
Comment (Mothblocks) [inline code/modules/unit_tests/simple_animal_freeze.dm:305]: mods

Comment (Fikou) [reply code/modules/unit_tests/simple_animal_freeze.dm:305]: OH MAINTAINERS!!!

**Comment by Jackal-boop** [comment]
Merge it

**Comment by IndieanaJones** [comment]
'ate obsessed
'ate paradox clone
luv' venom

**Comment by Shadowflame909** [comment]
speedmerge as the new end game for xenobio (remember changeling headslugs)

**Review by axietheaxolotl — COMMENTED** [review]
image
aww hell nah chief wat da hell is dis

File: code/modules/events/ghost_role/venom.dm [hunk 3]
```diff
@@ -0,0 +1,484 @@
...
+	shake_camera(hit_mob, 4, 3)
+	shake_camera(src, 2, 3)
+	venomify_mod(venom_target)
+
+/mob/living/simple_animal/hostile/venom/proc/venomify_mod(obj/item/mod/control/mod_target)
+	mod = mod_target
+	RegisterSignal(mod, COMSIG_MOD_ACTIVATE, PROC_REF(block_mod_activation))
+	RegisterSignal(mod, COMSIG_PARENT_QDELETING, PROC_REF(death))
+	ADD_TRAIT(mod, TRAIT_NODROP, REF(src))
+	ADD_TRAIT(mod.wearer, TRAIT_NOHUNGER, REF(src))
```
Comment (ArcaneMusic) [inline code/modules/events/ghost_role/venom.dm:176]: Curious, he claims it has no hunger, yet it has nohunger.

This says a lot about venom antag 2022.

**Comment by Fikou** [comment]
> image aww hell nah chief wat da hell is dis

make some better ones then

**Comment by moocowswag** [comment]
Its scifi-fantasy a pure black organic mass that absorbs all light touching it seems fitting (although not even nightmare does this with it`s sprite).

Also for some reason it reminds me of older pixelated horror games.

**Comment by StoneMason** [comment]
caption

**Comment by Bawhoppen** [comment]
I've never been so e

**Comment by Ekaterina-von-Russland** [comment]
Test merged on round 197307, broke TGUI, including but not limited to character preferences and latejoin.

**Comment by Addust** [comment]
you're all on crack

**Comment by SomeoneYouProbablyKnow** [comment]
Feedback from a round : While the antag itself is just a very funny shitpost the dynamic of 2 lads piloting a flesh automaton is a fun one. The potential for fun gameplay is very real.

**Comment by Fikou** [comment]
> Test merged on round 197307, broke TGUI, including but not limited to character preferences and latejoin.

no it didnt lol

**Comment by JohnFulpWillard** [comment]
Great Pull Request Fikou! Thank you for contributing to the /TG/Station Codebase!

**Comment by SomeoneYouProbablyKnow** [comment]
Highly based merge, depending in your timezone venom antag did come in 2022

**Comment by Jolly-66** [comment]
Modsuits will be REAL in 2023.

**Comment by Fikou** [comment]
> Great Pull Request Fikou! Thank you for contributing to the /TG/Station Codebase!

😄

**Comment by Bawhoppen** [comment]
Lol even for joke PRs, self-merges are RARE

**Comment by Addust** [comment]
WHY IS THIS MERGED WHAT IN THE NAME OF RATVAR HAVE YOU UNLEASHED UPON /TG/

**Comment by rellyreri** [comment]
YES!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!! MERRY CHRISTMAS!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!

**Comment by epicgamer545** [comment]
This is stupid
I love it

**Comment by IndieanaJones** [comment]
BESt NEW YEARS GIFT EVER

**Comment by MrDoomBringer** [comment]
This PR really makes me think Venom, adrenaline momentum And I'm not knowin' when I'm Ever gonna slow up and I'm Ready to snap any moment I'm Thinkin' it's time to go get 'em They ain't gonna know what hit 'em

**Comment by arnociiroc** [comment]
if venoms so good why isnt there a venom 2

**Pull request merged**